src/Microsoft.DotNet.Build.Tasks/GetPackageVersion.cs

```diff
@@ -34,7 +34,7 @@ public override bool Execute()
                 select el).FirstOrDefault();
 
             Debug.Assert(embeddedVerNode != null, "embeddedVerNode shouldn't be null");
-            PackageVersion = string.Format("{0}-{1}", embeddedVerNode.Value, DailyBuildNumber);
+            PackageVersion = string.Format("{0}-{1}", embeddedVerNode.Value, BuildNumber);
```

Comment:
That seems like it is going to be a problem for shipping stable releases. How do people ship a version of a package with just 1.0.0? Currently GetPackageVersion is used for all the places that produce the packages and it seems like we need a way to specify whether or not it is a prerelease or not. Perhaps we need another property or something in the NuSpec that indicates that.